packages/ai-jsx/src/core/opentelemetry.ts

```diff
@@ -59,8 +63,12 @@ export function openTelemetryStreamRenderer(streamRenderer: StreamRenderer): Str
           throw ex;
         } finally {
           if (result) {
+            const resultIsPartial = result.find(isElement);
             // Record the rendered value.
-            span.setAttribute('ai.jsx.result', result.find(isElement) ? debug(result, true) : result.join(''));
+            span.setAttribute('ai.jsx.result', resultIsPartial ? debug(result, true) : result.join(''));
+            if (!resultIsPartial) {
+              span.setAttribute('ai.jsx.result.tokenCount', getEncoder().encode(result.join('')).length);
+            }
```

Comment:
It seems like it would be nice if this and the standard logging were unified, such that we only had one place to attach metadata to render results.

@petersalas, I'm guessing your preferred means of that unification would be to kill the other logging. :smile:

Reply:
Yup exactly